make/modules/java.xml/Copy.gmk

```diff
@@ -24,14 +24,16 @@
 #
 
 include CopyCommon.gmk
+include Modules.gmk
 
 ################################################################################
+#
+# Copy property files from share/conf to CONF_DST_DIR
+#
+$(eval $(call SetupCopyFiles, COPY_XML_MODULE_CONF, \
+    DEST := $(CONF_DST_DIR), \
+    FILES := $(wildcard $(TOPDIR)/src/java.xml/share/conf/jaxp*.properties*), \
```

Comment:
I don't think you need, nor should have, the asterisk after the extension. You are only copying `.properties` files.

```suggestion
    FILES := $(wildcard $(TOPDIR)/src/java.xml/share/conf/jaxp*.properties), \
```

Reply:
Thanks Magnus!